src/views/TabView/TabView.js

```diff
@@ -127,6 +127,8 @@ class TabView extends PureComponent<void, Props, void> {
     );
   };
 
+  _renderPager = (props: *) => <TabViewPagerPan {...props} />;
```

Comment:
on android , i found there is still a bug, when i use ViewPagerAndroid with react-navigation . 
ViewPagerAndroid will not swipe normally , i found it is the problem of TabViewPagerPan , it is still use Animated.View transform:

`  style={[
          styles.sheet,
          width
            ? { width: routes.length * width, transform: [{ translateX }] }
            : null,
        ]}
        {...this._panResponder.panHandlers}
      >
        {Children.map(children, (child, i) => (
          <View
            key={navigationState.routes[i].key}
            testID={navigationState.routes[i].testID}
            style={
              width
                ? { width }
                : i === navigationState.index ? StyleSheet.absoluteFill : null
            }
          >
            {i === navigationState.index || width ? child : null}
          </View>
        ))}
      </Animated.View>`